### PR TITLE
README: updated

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+
+Control Tier Nodes: 
+- Mesos Master
+- Marathon
+- Exhibitor (for Zookeeper) 
+- Chronos
+- [`Flight Director`](https://github.com/adobe-platform/flight-director)
+
+Proxy Tier Nodes: 
+- [`CAPCOM`](https://github.com/adobe-platform/capcom) / Heatshield
```
Comment (iderdik) [inline README.md]: CAPCOM and Proxy run as a separate containers.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+Provides node-level services as [`Fleet Units`](https://coreos.com/using-coreos/clustering/) for every machine in the cluster.
+Most services (logging, metrics, monitoring) run on all nodes, some only run on specific tiers based on the metadata that is injected into Fleet.
+
+All Nodes:
+- Datadog (monitoring)
+- Sysdig (monitoring)
+- Sumologic (log collector)
+- Logrotate
+- Docker image cleanup
+- SSHD
```
Comment (chr0n1x) [inline README.md:13]: @iderdik this reminds me - did we hear back from the CoreOS dudes about this?

**Comment by chr0n1x** [comment]
:+1:

**Comment by iderdik** [comment]
:+1:

**Pull request merged**